features/developer_portal/admin/account/personal_details.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Dev Portal Buyer Personal Details
+  As a buyer
+  I want to change my personal details
+
+  Background:
+    Given a buyer logged in to a provider
+
+  # This is needed, otherwise assert_flash doesn't receive the right message.
+  # Check flash-buyer.js
+  @javascript
+  Scenario: Buyer doesn't use current password
+    Given the buyer wants to edit their personal details
+    When the buyer edits their personal details
+    Then they should not be able to edit their personal details
+
+  @javascript
+  Scenario: Buyer uses wrong current password
+    Given the buyer wants to edit their personal details
+    When the buyer edits their personal details
+    And the buyer writes a wrong current password
+    Then they should not be able to edit their personal details
+
+  @javascript
+  Scenario: Buyer uses correct current password
+    Given the buyer wants to edit their personal details
+    When the buyer edits their personal details
+    And they change their password
+    And the buyer writes a correct current password
+    Then they should be able to edit their personal details
+    And password must have changed
```

Comment:
Why the password is checked as a separate field? I think it should part of `they should/could be able to edit their personal details`.